src/engine.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
Whoa, that's a change. Probably better to wait with that change.

Reply:
Will you revert _this_ change to the solution file? We can't change the VS version, since current branch is not ready for it.